protected/views/patient/_130_diagnoses.php

```diff
@@ -40,6 +40,7 @@
 				<th>Diagnosis</th>
 				<?php if ($this->checkAccess('OprnEditOtherOphDiagnosis')) { ?>
 					<th>Actions</th>
+                    <th/>
```

Comment:
You might want to have an empty tag instead of a self-closing tag, this might not work on all browsers